picard/ui/metadatabox/__init__.py

```diff
@@ -137,8 +137,15 @@ def get_tag_name(self, index):
 
 class MetadataBox(QtWidgets.QTableWidget):
 
+    MIMETYPE_PICARD_TAGS = "application/vdr.picard"
+    MIMETYPE_TEXT = 'text/plain'
+    mimedatas = {
+        MIMETYPE_PICARD_TAGS: 'to_json',
+        MIMETYPE_TEXT: 'to_tsv',
+    }
+
     COLUMN_TAG = 0
-    COLUMN_ORIG = 1
+    COLUMN_OLD = 1
```

Comment:
This rename wasn't strictly needed.
My recent patch to tagdiff leads to this confusion: tagdiff uses old/new because it isn't really tied to anything. but this column is named after what it contains and its title (Original Tag).

Reply:
rgr that - I was following your lead, but you raise a good point re column names!